protocol/avdl/stellar1/local.avdl

```diff
@@ -353,4 +383,7 @@ protocol local {
 
   map<OutsideCurrencyCode, OutsideCurrencyDefinition> getAvailableLocalCurrencies();
   string formatLocalCurrencyString(string amount, OutsideCurrencyCode code);
+
+  void sendRequestCLILocal(string recipient, union { null, Asset } asset, 
```

Comment:
since `send` could be confused with sending money, what about changing this to `makeRequestCLILocal()`?